interceptor_test.go

```diff
@@ -81,22 +81,19 @@ func TestStreamingMetrics(t *testing.T) {
 						return now
 					}
 				}))),
-		}, []connect.ClientOption{}, happyPingServer())
-	stream := connectClient.CumSum(context.Background())
-	err := stream.Send(&pingv1.CumSumRequest{Number: 12})
-	if err != nil {
-		t.Error(err)
-	}
-	_, err = stream.Receive()
-	if err != nil {
-		t.Error(err)
+		}, []connect.ClientOption{}, okayPingServer())
```

Comment:
😢 The server used to be happy, but now it's just okay 😢

Reply:
We had a `happy`/`fail` server and I wanted to make the naming consistent so it felt worse to have a `sadPingServer()`... but I can fix this.

Reply:
Nah I'm just giving you a little grief :)